src/Stratis.Bitcoin.Networks/Policies/BitcoinStandardScriptsRegistry.cs

```diff
@@ -21,6 +21,7 @@ public class BitcoinStandardScriptsRegistry : StandardScriptsRegistry
             PayToPubkeyTemplate.Instance,
             PayToScriptHashTemplate.Instance,
             PayToMultiSigTemplate.Instance,
+            PayToFederationTemplate.Instance,
```

Comment:
Is it necessary to define this for this network?

Reply:
Removed.